com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeBrickPool.cs

```diff
@@ -354,13 +356,18 @@ static void SetPixel(DynamicArray<Color> data, int x, int y, int z, int dataLocW
             data[index] = value;
         }
 
-        public static void FillDataLocation(ref DataLocation loc, SphericalHarmonicsL2[] shl2, int startIndex, int count, ProbeVolumeSHBands bands)
+        public static void FillDataLocation(ref DataLocation loc, ProbeVolumeSHBands srcBands, NativeArray<Color> shData, int startIndex, int count, ProbeVolumeSHBands dstBands)
         {
+            var probeStride = srcBands == ProbeVolumeSHBands.SphericalHarmonicsL2 ? 7 : 3;
+
+            // Coeff constants that cancel out shader probe decoding
+            var zhalf = new Color(0f, 0f, 0f, 0.5f);
+            var half = new Color(0.5f, 0.5f, 0.5f, 0.5f);
+
```

Comment:
So instead of just writing black as default value, these constants cancel out shader decoding of probes data. This means you can load defaults without getting crazy results, or load L1 data into an L2 enabled config and things will still be correctly lit.

Reply:
Change is good for me, but why zhalf ? It's the 4th channel

Reply:
zero-half. lazy naming, I'll fix it.

Reply:
made these hopefully a bit more meaningful